calyx-opt/src/passes/compile_invoke.rs

```diff
@@ -287,6 +309,7 @@ impl Visitor for CompileInvoke {
         // Assigns representing the ref cell connections
         invoke_group.borrow_mut().assignments.extend(
             self.ref_cells_to_ports(Rc::clone(&s.comp), s.ref_cells.drain(..)),
+            // ), //the clone here is questionable? but lets things type check? Maybe change ref_cells_to_ports to expect a reference?
```

Comment:
The fact that it's an `Rc::clone` makes me feel less worried about it? That is, we're not cloning the underlying structure; we are just doing some reference-counting here.